.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Since this is a vs specific file, it should be removed from the PR.  You can add `.vscode` to the `.gitignore` file to prevent it from being checked in in the future.

Reply:
Thanks. I will do that immediately.